views/blog.html

```diff
@@ -20,43 +20,46 @@
 
 <div class="container mx-auto md:flex p-gutter">
 
-  <div class="md:w-2/3 md:mr-gutter lg:flex flex-wrap">
-    <!-- posts -->
-    {% for post in posts %}
-    {% if loop.index0 % 2 == 0 %}
-    {% set cls = "article-preview mb-gutter pb-gutter lg:w-1/2 lg:pr-gutter" %}
-    {% else %}
-    {% set cls = "article-preview mb-gutter pb-gutter lg:w-1/2 lg:pl-gutter" %}
-    {% endif %}
-    <div class="{{cls}}">
-      <a class="article-preview_image" href="/blog/{{post.slug}}">
-        <img class="article-preview_img" alt="{{ post.imageAlt or post.title or post.imageCaption }}" src="{{post.image or '/static/img/placeholder.svg'}}" />
-      </a>
-      <div class="article-preview_text">
-        <h1 class="article-preview_heading">
-          <a class="article-preview_heading-link hover:underline" href="/blog/{{post.slug}}">{{post.title | safe}}</a>
-        </h1>
-        <div class="mb-3">
-          <ul>
-            {% for category in post.categories %}
-            <li>
-              <a href="/blog?category={{category}}" class="inline px-3 py-1 text-xs text-white rounded mr-3 mb-3 bg-secondary" title="category">
-                {{ category }}
-              </a>
-            </li>
-            {% endfor %}
-          </ul>
-        </div>
-        <time class="article-preview_date">{{post.published}}</time>
-        <div class="article-preview_content">
-          {{ post.content | striptags | truncate(200) | safe }}
-          <p><a class="text-secondary hover:underline" href="/blog/{{post.slug}}">{{ __('Read full article') }}</a>
-            <p>
+  <div class="md:w-2/3 md:mr-gutter lg:flex flex-wrap justify-around">
+
+    <div class="flex flex-wrap">
+      <!-- posts -->
+      {% for post in posts %}
+      {% if loop.index0 % 2 == 0 %}
+      {% set cls = "article-preview mb-gutter pb-gutter lg:w-1/2 lg:pr-gutter" %}
+      {% else %}
+      {% set cls = "article-preview mb-gutter pb-gutter lg:w-1/2 lg:pl-gutter" %}
+      {% endif %}
+      <div class="{{cls}}">
```

Comment:
@EvgeniiaVak 
I was able to achieve the same layout as on prod by removing the `if` logic and using this piece of code instead
```
<div class="flex flex-wrap -mx-6">
      <!-- posts -->
      {% for post in posts %}
      <div class="article-preview mb-gutter pb-gutter lg:w-1/2 px-6">
        <a class="article-preview_image" href="/blog/{{post.slug}}">
...
```

Let me know if it looks good. I'll push the changes to this PR.

Reply:
@MuhammadIsmailShahzad looks super-cool!

Reply:
Will update once the changes are pushed.

Reply:
Changes pushed.
PR is updated and ready for review.